### PR TITLE
Actually register Collide State effect, and add Collidable state expression

### PR DESCRIPTION
**Pull request by DogLoverPink**
First off, I may be completely wrong, and I may have a made useless PR, but for me, the "Set Collidable" effect didn't work, and appeared to not be registered at all, and the "Collidable State" expression didn't work, and doesn't appear to exist in the code at all.

Some very slight syntax changes would be required for this to work:
- Changed "[set] collid(e|able) [state] [of] %entity% to %boolean%" **to** "[set] collid(e|able) [state] [of] %livingentity% to %boolean%"
- Changed "collid(e|able) [state] [of] %entity%" **to** "collid(e|able) [state] [of] %livingentity%"

Basically just changing entity to livingentity

### CODE REVIEW

**Comment by DogLoverPink** [comment]
Imma close this, as I don't really know what I'm doing.
